README.md

```diff
@@ -23,19 +23,19 @@ git clone --recursive git@github.com:cms-btv-pog/BTVNanoCommissioning.git
 # activate enviroment once you have coffea framework 
 conda activate btv_coffea
 ```
-### Coffea installation with Miniconda
-For installing Miniconda, see also https://hackmd.io/GkiNxag0TUmHnnCiqdND1Q#Local-or-remote
+### Coffea installation with Micromamba
+For installing Micromamba, see [[here](https://mamba.readthedocs.io/en/latest/installation/micromamba-installation.html)]
 ```
-wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+wget wget -L micro.mamba.pm/install.sh
```

Comment:
is the `wget` duplicated?

Reply:
Accidentally duplicated. It is a typo

Reply:
Thank you for changing this! will merge :)